The grep in xbps-src chroots does not support Perl regexes, so restore a prior
version of the grep to identify highest and lowest nodes in test/bind_range.

--- a/test/bind_range
+++ b/test/bind_range
@@ -92,8 +92,8 @@
 }
 
 HIGHESTCPU=$(ls /sys/bus/cpu/devices | sed s/cpu// | sort -n | tail -1)
-HIGHESTNODE=$(numactl -H | grep -Pzo 'node [0-9]* cpus: [0-9].*(.|\n)node [0-9]* size: [1-9].* MB' | tail -n1 | cut -f2 -d' ')
-LOWESTNODE=$(numactl -H | grep -Pzo 'node [0-9]* cpus: [0-9].*(.|\n)node [0-9]* size: [1-9].* MB' | head -n1 | cut -f2 -d' ')
+HIGHESTNODE=$(numactl -H | grep -e 'node [0-9]* cpus: [0-9]*' | tail -n1 | cut -f2 -d' ')
+LOWESTNODE=$(numactl -H | grep -e 'node [0-9]* cpus: [0-9]*' | head -n1 | cut -f2 -d' ')
 
 get_mask
 
